daemons/Makefile

```diff
@@ -28,6 +28,16 @@ deploy-serial: $(SERIAL_AWS_DAEMONS) $(SERIAL_GCP_DAEMONS)
 deploy-parallel: $(PARALLEL_AWS_DAEMONS) $(PARALLEL_GCP_DAEMONS)
 
 $(SERIAL_AWS_DAEMONS) $(PARALLEL_AWS_DAEMONS) scheduled-ci-build:
+	@for f in $@/*.tf; do \
+		echo $$f; \
+		if [ -e $$f ]; then \
+			rm -rf $@/.terraform/*.tfstate;\
+			./build_tf_deploy_config.py $@;\
+			(cd $@ ; terraform init);\
+			(cd $@ ; TF_VAR_daemon=$@ terraform apply -auto-approve);\
+		fi;\
+		break;\
```

Comment:
This means only the first .tf file will be applied. Did you mean it that way, or is this a leftover debug breakpoint?

Reply:
This was intentional: once the first .tf file is detected, we know we need to perform TF workflow.